tools/i18n_autogen.py

```
@@ -0,0 +1,195 @@
+modified = []
+added_keys = []
+for f, groups in per_file_literals.items():
+    module = f.stem
+    lang_dir = ROOT / 'lang' / module
+    lang_dir.mkdir(parents=True, exist_ok=True)
+    zhf = lang_dir / 'zh_CN.json'
+    zh = {}
+    if zhf.exists():
+        try:
+            zh = json.load(zhf.open(encoding='utf-8'))
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)